src/patternfly/patternfly-base.scss

```diff
@@ -0,0 +1,5 @@
+@import "./utilities/_all";
+@import "./_fonts";
+@import "./_icons";
```

Comment:
`@patternfly/react-core` will be using SVGs for icons and wont need the `./_icons`.  would it be possible to remove this from here and place it in the full version? Fonts should be ok to keep.

Reply:
@dmiller9911 that file is necessary to pull in the Font Awesome styling. Without it, the icons will not be available in anything but the full version.

Reply:
Ok. I will write something to remove this at build when we copy over the styles.